doc/install/install.md

```diff
@@ -130,6 +134,6 @@ spec:
 
 # Uninstall
 
-Run the command `make uninstall`.
+OLM can be uninstalled with the `operator-sdk olm uninstall` command. More information is available in the [operator-sd documentation](https://sdk.operatorframework.io/docs/cli/operator-sdk_olm_uninstall/)
 
-**NOTE** Valid just for local/manual installs. 
+**NOTE** local/manual installs can be removed with `make uninstall`.
```

Comment:
If we're interested in removing this target, does it make more sense to drop this section altogether?

Reply:
Probably makes sense. Will delete.